### PR TITLE
Dynamic UI themes

### LINKED ISSUE #101: Basic UI Themes: light, dark, cat

### PR DESCRIPTION
**Pull request by mattyg**
Resolves #101 

**Overview**
- add ui dependency `pinia-persistedstate-plugin`
- ui store for selecting, applying a theme and saving it to localstorage
- 3 example themes: Light, Dark & Cat
- user can apply a theme via their profile page

We might want to swap out the very absurd cat picture I found for another :laughing: 

**todo**
- [ ] search results text
- [ ] mew response button icons
- [ ] edit profile form text

### CODE REVIEW

**Comment by jost-s** [comment]
Cool feature!

The dark mode is too dark in some places IMO, for example when editing the profile
image


Or when looking at an agent profile
image

The "Reply" heading is faint too
image


The cat theme - well =D, cat image aside, readability suffers as well:
image

File: ui/src/components/MewList.vue [hunk 1]
```diff
@@ -34,4 +36,12 @@ defineProps({
     required: true,
   },
 });
+
+const onMewClick = (actionHash: Uint8Array) => {
+  console.log('going!');
```
Comment (jost-s) [inline ui/src/components/MewList.vue:41]:
```suggestion
```

**Review by jost-s — CHANGES_REQUESTED** [review]
Only cosmetic changes requested

**Comment by mattyg** [comment]
Since we're leaning towards #65 which will superseed this ticket, and it feels lower priority for now. I'm going to close this PR.

**Pull request closed without merging**